client/document/404.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Head from '../components/head';
+
+class NotFound extends Component {
+	render() {
```

Comment:
If the component only has a `render` method, we should turn it into a `PureComponent`, or better yet, a stateless functional one 😬